dcos/marathon.py

```diff
@@ -130,6 +130,8 @@ class Client(object):
     """
 
     def __init__(self, marathon_url, timeout=http.DEFAULT_TIMEOUT):
+        if not marathon_url.endswith('/'):
+            marathon_url += '/'
         self._base_url = marathon_url
```

Comment:
After longer discussion on slack we agreed that for now we will implement this minimalistic patch and later we can try to fix it globally by first tracking all the places that use urljoin and then making sure that correct behavior will not brake user's tools.